pxr/usd/usdLux/lightAPI.h

```diff
@@ -384,7 +555,17 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // COLOR 
     // --------------------------------------------------------------------- //
-    /// The color of emitted light, in energy-linear terms.
+    /// The color of emitted light, in the rendering color space.
+    /// 
+    /// This color is just multiplied with the emission. In the case of a
```

Comment:
Again, the suggestion here is to phrase this section to bias towards avoiding the need to uplift in the first place. We are better served to gently steer people away from wobbly-wobbly bits and towards methods based on measurable colorimetry.

Reply:
@anderslanglands @meshula  - suggested change here:

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R560-R569

Reply:
In this case I think the original wording was good. You _have_ to uplift an RGB in order to render spectrally.

Reply:
Restored the instructions about uplifting, but also kept the recommendation to use a "well defined" color space:

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/e493051f2d48654482586cbc32d46eb3a84b930d#diff-ec32e12ccc5be018c3a056d787ed4acdddd0653b33d1cc95593a8e5c26a1a75fR85-R89